### ISSUE TITLE
Null termination of VERSIONINFO strings?

### ISSUE THREAD

**Issue by MaulingMonkey** [mxre/winres#25]
**TL;DR:** Might be worth inserting `\0` after the escaped `v` on line 400 here:
https://github.com/mxre/winres/blob/82f55f8b46ca7ac44971b6f14b5770ab5d65d8df/lib.rs#L396-L404

[MSDN Examples](https://docs.microsoft.com/en-us/windows/win32/menurc/versioninfo-resource#examples) for VERSIONINFO resources appear to explicitly null terminate file/product version strings (`VER_FILEVERSION_STR` / `VER_PRODUCTVERSION_STR`).  The docs aren't super clear on if this is actually necessary though.  Other documentation is muddled:  [STRINGTABLE resource](https://docs.microsoft.com/en-us/windows/win32/menurc/stringtable-resource)s appear to be implicitly null terminated... but [User-Defined Resource](https://docs.microsoft.com/en-us/windows/win32/menurc/user-defined-resource)s explicitly aren't: "RC does not automatically append a terminating null character to a string."  I'd mostly worry about mixed Rust/C(++) codebases possibly reading back the version strings.  E.g. [LoadStringA](https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-loadstringa)(hinstance, id, &ptr, 0) will give you C-string pointer - not sure if you can get C-string pointers directly from Win32 for the version strings?
